### DISCUSSION TITLE
Input-number @chnage didn't trigger when reset value to null

### DISCUSSION THREAD

**Discussion by yuniit** [element-plus/element-plus#11442] [Q&A] [answered]
Hello, I have a problem with el-input-number when I reset all value the @change  event didn't trigger.

[see demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtaW5wdXQtbnVtYmVyIHYtbW9kZWw9XCJudW1cIiAgQGNoYW5nZT1cImhhbmRsZUNoYW5nZVwiIC8+XG4gIDxicj5cbiAgPGJyPlxuICBAY2hhbmdlIHRyaWdnZXIgY291bnQgOiB7e3RyaWdnZXJDb3VudH19XG4gIDxicj5cbiAgPGJyPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgbnVtID0gcmVmKDEpXG5jb25zdCB0cmlnZ2VyQ291bnQgPSByZWYoMClcblxuY29uc3QgaGFuZGxlQ2hhbmdlID0gKHZhbHVlOiBudW1iZXIpID0+IHtcbiAgdHJpZ2dlckNvdW50LnZhbHVlKytcbiAgY29uc29sZS5sb2coJ3ZhbHVlICcsIHZhbHVlKVxufVxuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

https://user-images.githubusercontent.com/30621352/217149349-9989c067-455d-4c90-97c3-81655a6806a7.mov

**Comment by markhermano** [comment, marked as answer]
Add the new property `value-on-clear` which is available in `v2.2.0` onwards.

```html
<el-input-number v-model="num"  @change="handleChange" :value-on-clear="0" />
```
This way it will trigger the change when manually clearing the input.
